app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/pebble/PebbleDisplayStandard.java

```diff
@@ -23,10 +26,18 @@
 
     private static double last_time_seen = 0;
 
+    private String StatusLine="";
+
 
     @Override
     public void startDeviceCommand() {
-        sendData();
+        Handler handler = new Handler();
```

Comment:
Can you explain what this delay is for?

Reply:
This delay is necessary to display the latest loop status. AAPS needs some connection and action time for setting treatment data. Without delay, outdated (5min) loop data will be shown on the watchface. The delay time depends on connection speed of pump model. Dana R is faster than Combo and i think 20sec should be enough. For Combo users maybe 30-40sec would be necessary. The best solution would be to send a new packet to the pebble right after the StatusLine has been updated.